en/docs/references/concepts/authentication/hybrid-client-profile.md

```diff
@@ -45,29 +39,29 @@ token, refresh token and ID token.
 The following curl command is used to request tokens from the token endpoint:
 
 ``` java
-curl -k -v — user <Client ID>:<Client secret> -d “grant_type=authorization_code&code=99b34587–5483–374d-8b25–50485498e761&redirect_uri=<callback_url>" <token_endpoint>
+curl -k -v — user <client_id>:<client_secret> -d "grant_type=authorization_code&code=99b34587–5483–374d-8b25–50485498e761&redirect_uri=<callback_url>" <token_endpoint>
```

Comment:
Should we start giving the placeholders in capital letters? If you feel it's better to do it on the go, please check other places in this PR too.